data/supporters.json

```diff
@@ -1045,7 +1034,17 @@
           "phone": "+49 151 22962186",
           "email": "daniel@dmbch.net",
           "twitter": "dmbch"
-        }
+        },
+        {
+          "name": "Angela Galliat",
+          "email": "agalliat@galliat.de",
+          "twitter": "agalliat"
+        },
+        {
+          "name": "Tobias Weinert",
+          "phone": "+49 179 2162979",
+          "twitter": "tweini"
+        },
```

Comment:
Could you please fix this parsing error (trailing comma) in the JSON file and check? Thanks!